incubator/docker-registry/Chart.yaml

```diff
@@ -8,3 +8,5 @@ keywords:
 maintainers:
 - name: sbezverk
   email: sbezverk@cisco.com
+- name: Andrey Arapov
+  email: andrey.arapov@nixaid.com
```

Comment:
Can you clarify why you are adding yourself as a maintainer?

Reply:
I have a wish to be. :-)
Would this be possible?

Reply:
OK, I'll leave you in. You will have to review future changes.